remoting/src/main/java/org/apache/rocketmq/remoting/common/RemotingUtil.java

```diff
@@ -99,6 +99,10 @@ public static String getLocalAddress() {
             ArrayList<String> ipv6Result = new ArrayList<String>();
             while (enumeration.hasMoreElements()) {
                 final NetworkInterface networkInterface = enumeration.nextElement();
+                if ("docker0".equals(networkInterface.getName())) {
```

Comment:
docker1 could also exist? docker bridge is a configuration option in docker architecture. could we have a common way to sweep out these ips?

Reply:
Hi, we can read the sysfs directory to check if the interface is a bridge. This method is also used in some bridge tools, such as `brctl`, in fact, I was inspired by this tool.

Reply:
more explantion:
http://man7.org/linux/man-pages/man5/sysfs.5.html
https://unix.stackexchange.com/questions/40560/how-to-know-if-a-network-interface-is-tap-tun-bridge-or-physical
https://github.com/Gandi/bridge-utils/blob/master/libbridge/libbridge_init.c